tests/integration/routes/accounts/patch-accounts-test.js

```diff
@@ -97,8 +98,40 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not found', {todo: true}, function (t) {
-      t.end()
+    group.test('Not found', function (t) {
+      var couchdb = nock('http://localhost:5984')
+          .get('/_users/_design/byId/_view/byId')
+          .query({
+            key: '"xyz1234"',
+            include_docs: true
+          })
+          .reply(200, {
+            total_rows: 1,
+            offset: 0,
+            rows: []
+          })
+
+      server.inject({
+        method: 'PATCH',
+        url: '/accounts/xyz1234',
+        headers: cloneDeep(routeOptions.headers),
```

Comment:
Thanks, wasn't sure if there was any extra stuff going on, so did a clone to be cautious. I've changed this to a straight assignment.